### PR TITLE
Module 1 Capstone

### PR DESCRIPTION
**Pull request by GuilhermeGiachellin**
-The project was maintained in the same format as the template.
-Minor changes were made to accommodate the subject.
-Both versions were developed in this branch.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    <header>
+        <img src="./assets/greekhamburguer.png" alt="hamburguer" id="hamburguer" class="hamburguer">
+        <a href="./index.html"><img src="./assets/Illustration.png" alt="Logo" class="logo"></a>
+        <nav>
+            <ul class="desktop_nav">
+                <li><a href="./index.html">Join</a></li>
+                <li><a href="./pages/about.html">About</a></li>
+                <li><a href="./index.html">Program</a></li>
+            </ul>
+        </nav>
```
Comment (henatan99) [inline index.html]: - [x] The navbar (nav element) should have its background transparent or similar to the top section background. For your reference compare the attached screenshots in your and original designs. 

image
image

- [x] The navbar has two sections in the Desktop view in the original design. Please see the attached screenshots for comparison and make the appropriate change. 
image
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        </ul>
+    </section>    
+    <footer>
+        <h2>Partners</h2>
+        <hr>
+        <div class="partners_cnt">
+            <img src="./assets/youtube.png" alt="youtube">            
+            <img src="./assets/google.png" alt="google">
+        </div>
+    </footer>
```
Comment (henatan99) [inline index.html]: - [x] You forgot to add the footer section lower part which shows the logo and copyright text. Please see the attached screenshot for the original design.
image
- You may also need to make some typography changes in the Partners section title.

File: js/dynamic_speakers.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  imagePath: './assets/hercules.jpg',
+  name: 'Hercules',
+  title: 'Son of Zeus',
+  description: 'Completed the 12 jobs became king, was tricked by Hera',
+}, {
+  imagePath: './assets/perseus.jpg',
+  name: 'Perseus',
+  title: 'Son of Zeus',
+  description: 'Killer of Medusa and used her head to save Argos',
+}];
```
Comment (henatan99) [inline js/dynamic_speakers.js]: - [x] The speakerInfo array should have 6 objects as in the original design. 
 
image

File: style.css [hunk 4]
```diff
@@ -0,0 +1,783 @@
...
+  width: fit-content;
+  margin: 5vh auto 0 auto;
+}
+
+.program {
+  display: flex;
+  justify-content: space-between;
+  width: 90vw;
+  height: 120px;
+  background: rgba(211, 211, 211, 0.5);
```
Comment (henatan99) [inline style.css]: - [x] You should consider some changes in your styling here as your design has some issues in spacing and contrast compared to the original design. 
image
image

File: style.css [hunk 5]
```diff
@@ -0,0 +1,783 @@
...
+}
+
+.speaker_list {
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+}
+
+.speakers {
+  display: flex;
```
Comment (henatan99) [inline style.css:174]: - [x] You also need to make some changes to the styling here so your design looks very close to the original design. Please see the attached screenshot for reference. 
image
image

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @GuilhermeGiachellin , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._
- [x] Your readme should include an acknowledgment that gives appropriate credit to the author of the original design of your project and also any other source you have used. 
- [x] Your readme should include a brief description of the project. The link to the screenshot image is also broken.  
- [x] I haven't seen a video link either in your PR summary or the readme file. Kindly add the video link.  
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by smcommits — APPROVED** [review]
### Status: Approved 👍🏽 

Hi @GuilhermeGiachellin, 

## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**